test/integration/podman_test.go

```diff
@@ -1,9 +1,20 @@
 package integration
 
-// This was commented out until we can confirm the new behavior of the github runner
-// tests started throwing "read: connection reset by peer" when connecting to ssh://root@localhost:2222/run/podman/podman.sock
-// we might need to think of another creative way to test this, but in the meantime it has been failing stereoscope builds
-/*
```

Comment:
note that all of the below code was commented out on main (hard to tell in the diff)